### PR TITLE
Adding button to promote or demote the user

### PR DESCRIPTION
**Pull request by BishoyHanna91**
I added an option  when the user is already validated; to  promote a user to be manager and demote the manager to be user.
Please, check the code.

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -196,39 +196,73 @@ module.exports = function (passport) {
...
+    res.redirect('/userManagement');
+  });
+
+
+  router.post('/userManagement/demoteManager/:badge',checkAuth, function (req, res) {
+    if (!req.body) {
+      return res.sendStatus(400);
+    }
+    var BagdeID = req.params.badge;
+    dbAPI.modifyUser(BagdeID, { status: "User" }).then(function (result) {
```
Comment (danieleynis) [inline routes/index.js]: Before modifying this should check first that:
1) The user we are modifying actually has the status 'Manager'
2) Is also not an 'Admin'

File: routes/index.js [hunk 2]
```diff
@@ -196,39 +196,73 @@ module.exports = function (passport) {
   });
 
 
-  router.get('/userManagement/:badge', checkAuth, function(req,res){
+  router.post('/userManagement/promoteUser/:badge',checkAuth, function (req, res) {
+    if (!req.body) {
+      return res.sendStatus(400);
+    }
+    var BagdeID = req.params.badge;
+    dbAPI.modifyUser(BagdeID, { status: "Manager" }).then(function (result) {
```
Comment (danieleynis) [inline routes/index.js]: Before modifying this should check first that:
1. The user we are modifying actually has the status 'User'
2. Is also not an 'Admin'

File: routes/index.js (hunk 1 above)
Comment (BishoyHanna91) [reply routes/index.js]: I do not know is there a way to get a specific data of the user as status ?

Comment (danieleynis) [reply routes/index.js]: Use `validateUser()`?

Comment (BishoyHanna91) [reply routes/index.js]: I guess we do not need these extra as it is already checked in the UI, if you go to an admin you won't see any of these options.

Comment (danieleynis) [reply routes/index.js]: That is true. But that API route could potentially be reused somewhere else and we need to ensure that no assumptions can be made. Doing that check again makes it more secure.

Comment (BishoyHanna91) [reply routes/index.js]: I am almost done with it, I am just doing some more tests to make sure everything is going well.

Comment (danieleynis) [reply routes/index.js]: Sounds good. Thanks.

**Pull request merged**